posts/2024-06-20-types-team-update.md

```diff
@@ -0,0 +1,225 @@
+users to encounter these bugs by accident. Regardless, we deeply care about fixing them
+and are working towards a fully sound and ideally verified type system.
+
+### The type system should be Consistent
+
+The type system should be easy to reason about. We should avoid rough edges and
+special-cases if possible. We want to keep both the implementation and user-facing behavior
+as simple as possible. Where possible we want to consider the overall design instead of
+providing local targeted fixes. This is necessary to build trust in the soundness of the
+type system and allows for a simpler mental model of Rust.
+
+### The type system should be Extensible
+
+Rust is still evolving and we will be required to extend the type system to enable new
+language features going forward. This requires the type system to be extensible and
+approachable. The design of the language should not be adapted to work around
+short-comings of its current type system implementation. We should collaborate with
+other teams and users to make sure we're aware of their problems and consider possible
+future extensions in our implementation and design.
+
+### The type system should be Fast
+
+We care about the compile times of Rust and want to consider the impact on compile times
+of our designs. We should look for effective approaches to speed up the existing implementation,
+by improving caching or adding fast paths where applicable. We should also be aware of the
+compile time impact of future additions to the type system and suggest more performant
+solutions where possible.
+
+## Updates
+
+We have been very active over the last year and made some significant progress. There
+are also a few non-technical updates we would like to share.
+
+## Organizational updates
+
+First, a huge welcome to the two new members to team since the announcement post: [@BoxyUwU] and [@aliemjay]. [@BoxyUwU] has been doing a lot of work on const generics and made significant contributions to the design of the next generation trait solver. [@aliemjay] has been working on some very subtle improvements to opaque types - `impl Trait` - and to borrow checking. They are both invaluable additions to the team.
+
+We also organized another in-person Types Team meetup last October, immediately prior to EuroRust. We discussed the state of the team, looked at current implementation challenges and in-progress work, and reviewed and updated [the roadmap from the previous meetup][PrevRoadmap]. Most of this will be covered in this blog post.
+
+Finally, as discussed in the [RFC](https://rust-lang.github.io/rfcs/3254-types-team.html), we would like to have leads rotate out regularly, largely to help share the burden and experience of leads' work. So with that being said, [@nikomatsakis](https://github.com/nikomatsakis) is rotating out and [@lcnr](https://github.com/lcnr) is joining to co-lead alongside [@jackh726](https://github.com/jackh726/).
+
+## Roadmap progress and major milestones
```

Comment:
Another major accomplishment that could be worth mentioning is:

- https://github.com/rust-lang/rust/pull/116733

That seems on-par in terms of significance with some of the other things mentioned.

Reply:
while this is a very nice and necessary change, it does feel slightly "smaller in scope" to me than the other opaque type changes mentioned here. tbh I also just don't know how I would best add that to the existing document.